tests/01-status/input/ft-sync.p1

```diff
@@ -8,4 +8,4 @@ collection: testuser/index
 
 -- ignored:
 
-*.py
\ No newline at end of file
+*.py
```

Comment:
This is not yet implemented/documented, for now remove the `-- ignored:` feature. Be ruthless about deleting things, don't just keep because of sentimental reasons. As of now we do not support ignored.